<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccessDialog</class>
 <widget class="QMainWindow" name="AccessDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>486</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>KlapKlap</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="central_widget">
   <widget class="QLabel" name="gif_viewer">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>110</y>
      <width>280</width>
      <height>270</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="access_frame">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>320</width>
      <height>471</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <layout class="QVBoxLayout" name="loginLayout_5">
     <item>
      <widget class="QWidget" name="signup_widget" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="loginLayout_3">
        <item>
         <widget class="QLabel" name="title_label">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Crea un nuovo account</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="subtitle_label">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>E scopri il nostro fantastico editor di condivisione in tempo reale</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="signup_form" native="true">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <layout class="QVBoxLayout" name="loginLayout_4">
           <item>
            <widget class="QLabel" name="signup_name_label">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string extracomment="Inserisci il tuo nome"/>
             </property>
             <property name="text">
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="signup_name_input">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>22</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string extracomment="Inserisci il tuo nome"/>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Normal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="signup_surname_label">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string extracomment="Inserisci il tuo cognome"/>
             </property>
             <property name="text">
              <string>Surname</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="signup_surname_input">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>22</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string extracomment="Inserisci il tuo cognome"/>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Normal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="signup_email_label">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Email</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="signup_email_input">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>22</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string extracomment="Insersci la tua password. Deve contenere almeno 8 caratteri"/>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Normal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="signup_username_label">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Username</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="signup_username_input">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>22</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string extracomment="Inserisci il tuo username"/>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Normal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="signup_password_label">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Password</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="signup_password_input">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>22</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string extracomment="Inserisci la tua email"/>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="signup_hint_label">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="signup_btn">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #FFF;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(95, 160, 255, 246));
border: 1px solid white;</string>
             </property>
             <property name="text">
              <string>Registrati</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCommandLinkButton" name="back_link_btn">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>25</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string notr="true">Torna al login</string>
             </property>
             <property name="iconSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="description">
              <string notr="true"/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="login_widget" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="loginLayout_2">
        <item>
         <widget class="QLabel" name="logo_view">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="login_form" native="true">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <layout class="QVBoxLayout" name="loginLayout">
           <item>
            <widget class="QLabel" name="login_email_label">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Username</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="login_username_input">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>22</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string extracomment="Inserisci la tua email"/>
             </property>
             <property name="toolTipDuration">
              <number>2</number>
             </property>
             <property name="text">
              <string>klaus.cuko</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="login_hint_label">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="login_password_label">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Password</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="login_password_input">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>22</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string extracomment="Inserisci la tua password"/>
             </property>
             <property name="toolTipDuration">
              <number>2</number>
             </property>
             <property name="text">
              <string>Campioni37</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="login_hint_label_2">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="login_btn">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>25</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="toolTipDuration">
              <number>-1</number>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #FFF;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(232, 79, 59, 246));
border: 1px solid white;</string>
             </property>
             <property name="text">
              <string>Login</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="choise_label">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>22</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>oppure</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="show_signup_btn">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>25</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #FFF;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(95, 160, 255, 246));
border: 1px solid white;</string>
             </property>
             <property name="text">
              <string>Registrati</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
